packages/gatsby-image/src/index.js

```diff
@@ -131,6 +131,14 @@ function groupByMedia(imageVariants) {
     (variant.media ? withMedia : without).push(variant)
   )
 
+  if (without.length > 1 && process.env.NODE_ENV !== `production`) {
+    console.warn(
+      `We've found ${
+        without.length
+      } sources without a media property. You should only have 1.`
```

Comment:
"You should only have 1", isn't clear. Only 1 property with or without media property? It confused me for a little bit at first by what was meant.

Is referring to them as `sources` clear enough? I see on the docs for the PR there's a `sources` variable, so I guess so. `image variants` made a bit more sense to me, since there are multiple sources in the picture element to support additional image formats, and multiple images of different sizes for srcset.

What should a user do with this information? Do they know which source can exclude a media prop/key? Would linking to relevant doc page/header help?

Reply:
Thanks!